File: app/models/covid_data.rb
class CovidData < ApplicationRecord
end

# == Schema Information
#
# Table name: covid_data
#
#  id               :bigint           not null, primary key
#  fips             :text             default("Unknown"), not null
#  date             :date             not null
#  county           :text             default("Unknown"), not null
#  state            :text             default("Unknown"), not null
#  cases            :integer          default(0)
#  deaths           :integer          default(0)
#  confirmed_cases  :integer          default(0)
#  confirmed_deaths :integer          default(0)
#  probable_cases   :integer          default(0)
#  probable_deaths  :integer          default(0)
#  created_at       :datetime         not null
#  updated_at       :datetime         not null
#

File: client/src/models/CovidData.ts
import { Model, attribute as attr, initialize, STRING, NUMBER } from '../lib/Model'

export class CovidData extends Model {
  static modelName = 'covidData'

  constructor(data?: any) {
    super()
    initialize(this, data)
  }

  @attr({ type: STRING })
  fips: string = ''

  @attr({ type: STRING })
  state: string = ''

  @attr({ type: STRING })
  county: string = ''

  @attr({ type: NUMBER })
  cases: number | null = null

  @attr({ type: NUMBER })
  deaths: number | null = null

  // @attr({ type: NUMBER })
  // confirmed_cases: number | null = null

  // @attr({ type: NUMBER })
  // confirmed_deaths: number | null = null

  // @attr({ type: NUMBER })
  // probable_cases: number | null = null

  // @attr({ type: NUMBER })
  // probable_deaths: number | null = null
}
